exhale/graph.py

```diff
@@ -198,6 +198,8 @@ def __init__(self, name, kind, refid):
             self.return_type = None # string (void, int, etc)
             self.parameters = [] # list of strings: ["int", "int"] for foo(int x, int y)
             self.template = None # list of strings
+        if self.kind in ["class", "struct"]:
+            self.template = [] # list of strings
```

Comment:
TODO: use a set, `in {"class", "struct"}`

Question: do we not also want this for `"function"`?  Currently unused, but seems likely down the road.

Reply:
Removed entirely, it's not used.